app.py

```diff
@@ -6,17 +6,21 @@
 
 from producer import Producer
 
-
 load_dotenv()
 
 REDIS_HOST = '127.0.0.1'
```

Comment:
should be coming from env variables, see 12 factor app book https://12factor.net/config